src/apscheduler/job.py

```diff
@@ -145,7 +153,7 @@ def _get_run_times(self, now):
         """
         run_times = []
         next_run_time = self.next_run_time
-        while next_run_time and next_run_time <= now:
+        while next_run_time and next_run_time.astimezone(UTC) <= now.astimezone(UTC):
```

Comment:
Why is this necessary? datetime comparisons take the fold attribute into account.